src/main/java/com/itwins/foreverbio/models/Facture.java

```diff
@@ -0,0 +1,95 @@
+package com.itwins.foreverbio.models;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.Map;
+
+@Entity
+@Table(name="facture")
+public class Facture {
+
+    @Id
+    private int id;
+    private String date;
+
+    //private String produit;
```

Comment:
Next time we will use a list of ligne de commande object that will contain a product ,  qte and total.